stores/upstash/src/storage/index.ts

```diff
@@ -659,19 +659,26 @@ export class UpstashStore extends MastraStorage {
     } = { namespace: 'workflows' },
   ): Promise<WorkflowRuns> {
     try {
+      const actualNamespace = namespace || 'workflows';
```

Comment:
at line 644 do this `namespace = 'worfklows'`

```suggestion
```

This allow you to keep namespace everywhere below